src/mds/mdstypes.h

```diff
@@ -1137,7 +247,7 @@ inline void decode(old_inode_t<Allocator> &c, ::ceph::buffer::list::const_iterat
 }
 
 /*
- * like an inode, but for a dir frag 
+ * like an inode, but for a dir frag
```

Comment:
Please undo these whitespace changes. (Use `git reset -p ...`)

Reply:
I'll do it.

Reply:
I still see these whitespace changes?

Reply:
It was fixed in the last commit

Reply:
Done